Database/Repositories/Interfaces/ITournamentsRepository.cs

```diff
@@ -68,6 +68,13 @@ Task<ICollection<Tournament>> GetAsync(int page, int pageSize, TournamentQuerySo
     /// <returns>The updated <see cref="Tournament"/></returns>
     Task<Tournament?> AcceptPreVerificationStatusesAsync(int id);
 
+    /// <summary>
+    /// Reruns automation checks for a tournament and all child entities
+    /// </summary>
+    /// <param name="id">Tournament id</param>
+    /// <param name="force">Whether to overwrite fully Verified/Rejected data (dangerous)</param>
+    Task RerunAutomationChecks(int id, bool force = false);
```

Comment:
Method name and documentation is quite misleading imo. This method isn't actually running any automation checks, it is more appropriate to say that it is "signaling to the data worker that automation checks should be run again". Interpret that how you will but I think it should at the very least better describe what it is doing in documentation if not method name

Reply:
agreed